pkgs/tools/archivers/trrntzip/default.nix

```diff
@@ -0,0 +1,52 @@
+# This is a revival of the old trrntzip at
+# https://sourceforge.net/projects/trrntzip
+#
+# See https://sourceforge.net/p/trrntzip/discussion/457469/thread/d3610ea3b6/
+# there hasn't been any response
+#
+# Besides the new one is on github instead of sourceforge
+# which makes life for us easier
+
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "trrntzip";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "0-wiz-0";
+    repo = "trrntzip";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-EqGvP1HbO2PhSoYelhTY0edfwDg2W6LtjoijCoLL5a4=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ zlib ];
```

Comment:
Looks like there's some optionals features involving bzip2: https://github.com/0-wiz-0/trrntzip/blob/29d141d02bb677f593b76cf5fead59ef6056eb3d/src/contrib/zip.c#L1003

Not sure how important it is.

Reply:
I have no idea. I don't see why bzip2 would be needed when this is supposed to be a package for zip files.

Reply:
Seems to be a supported compression mode. But I don't see it reflected in the CMAKE files. Maybe we can just leave it be